chihacknight/state-by-state-decarbonization
# Move to State Icons, Add Lato Font, & Improve Overall Styling

## Overview

Adds state icons for each state, moves to the Lato font, and does some overall styling tweaks/improvements. Here's a diff:

| Before | After |
| --- | --- |
| ![Screenshot from 2022-03-22 21-16-11](https://user-images.githubusercontent.com/3187531/159610689-75d59f77-5091-482e-acdb-b93991cb25ae.png) | ![Screenshot from 2022-03-22 21-16-20](https://user-images.githubusercontent.com/3187531/159610688-0b122d0f-921c-4262-9732-4b317ceed92b.png) |
| ![Screenshot from 2022-03-22 21-16-28](https://user-images.githubusercontent.com/3187531/159610687-63c3a95b-0d69-4620-b2be-06394c3f7121.png) | ![Screenshot from 2022-03-22 21-16-34](https://user-images.githubusercontent.com/3187531/159610685-1191edbe-9574-4ae1-a656-843273cba813.png) |

Closes #37.

### Demo

See Overview.

### Notes

I just picked a font after trying out a few, I think this one looks solid! As for my other stylistic tweaks I was generally making things more readable and closer to the mocks.

## Testing Instructions

* Checkout to branch and confirm homepage looks good and state details page looks good!
